default.nix

```diff
@@ -1,5 +1,169 @@
-{ lib ? (import <nixpkgs> {}).lib }: {
-  inherit (import ./lib {
-    inherit lib;
-  }) config create mount;
+{ lib ? <nixpkgs/lib> }:
```

Comment:
```suggestion
{ lib ? import <nixpkgs/lib> }:
```

Reply:
I think you don't really need importing to be able to use the lib.